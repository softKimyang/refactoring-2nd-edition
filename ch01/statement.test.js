import plays from './data/plays.js'
import invoices from './data/invoices.js'
import html from './statement.js'

const invoice = invoices[0];
const expected = `청구 내역 (고객명: BigCo)
hamlet: $650.00(55)석
As You Like It: $580.00(35)석
Othello: $500.00(40)석
총액: $1,730.00
적립 포인트: 47점`;

test('리팩토링 전 테스트 ', () =>{
  expect(html(invoice, plays)).toMatchSnapshot();
});
