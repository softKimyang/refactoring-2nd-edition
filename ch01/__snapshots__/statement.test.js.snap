// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`리팩토링 전 테스트  1`] = `
"청구 내역 (고객명: BigCo)
hamlet: $650.00(55)석
As You Like It: $580.00(35)석
Othello: $500.00(40)석
총액: $1,730.00
적립 포인트: 47점
"
`;
